job_board/tests.py

```diff
@@ -27,27 +26,22 @@ def test_data_creation(self):
 
         assert Location.objects.filter(name='Eilat', latitude=29.559191910075217, longitude=34.95283437084102).exists()
 
-        assert YearsOfExperience.objects.filter(text='5-10 years').exists()
-
         assert Preference.objects.filter(
             job_type=JobType.objects.first(),
             location=Location.objects.first(),
-            years_of_experience=YearsOfExperience.objects.first()).exists()
+            years_of_experience=Preference.yearsOfExperience.YEARS5ANDABOVE,
+            work_schedule=Preference.workSchedule.FULLTIME).exists()
 
     def test_data_deletion(self, create_data_models_instances):
         job_type = create_data_models_instances[0]
```

Comment:
As @Yarboa instructed me at #64 filter through DB and match to fixture attributes, for example:
```py
job_type = JobType.objects.filter(text=create_data_models_instances[0].text).first()
```
and apply to location and preference

Reply:
This one still needs to be applied to lines 43,44,45

Reply:
After @paOmer's notes will be applied, approve